futurex_openedx_extensions/helpers/admin.py

```diff
@@ -312,6 +320,48 @@ class DataExportTaskAdmin(admin.ModelAdmin):
     search_fields = ('filename', 'user__email', 'user__username', 'notes')
 
 
+class ConfigAccessControlForm(forms.ModelForm):
+    """Admin class of ConfigAccessControl model"""
+
+    class Meta:
+        model = ConfigAccessControl
+        fields = '__all__'
+
+    def clean_path(self) -> str:
+        """Validates path with default tenant config."""
+        path = self.data['path']   # pylint: disable=redefined-outer-name
```

Comment:
this pylint quality check must not be ignored unless it's a test fixture
```suggestion
        key_path = self.data['path']
```